mrmustard/lab_dev/circuits.py

```diff
@@ -427,6 +430,45 @@ def validate_path(self, path) -> None:
 
             remaining[i1] = (remaining[i1] @ remaining.pop(i2))[0]
 
+    def serialize(self, filestem=None):
+        r"""
+        Serialize a Circuit.
+
+        Keyword Args:
```

Comment:
```suggestion
        Args:
```
This should be moved to the bottom of the docstring for consistency